index.html

```diff
@@ -8,6 +8,70 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h2 class="title">Front-End Tools</h2> 
+  <div class="tools"> 
+    <div class= "tools-section"> 
```

Comment:
I think that this tag isn't necessary. I recommend you to do a structure in the HTML more simple